lib/pages/settings_dashboard/settings/settings.dart

```diff
@@ -102,10 +131,15 @@ class SettingsController extends State<Settings> with ConnectPageMixin {
           }
           await matrix.client.logout();
         } catch (e) {
-          Logs().e('SettingsController()::logoutAction - error: $e');
+          Logs().e('SettingsController()::_logoutActionsOnWeb - error: $e');
         } finally {
           if (PlatformInfos.isWeb) {
```

Comment:
IMO no need to add the condition since on desktop the log out process is the same as on web

Reply:
Updated it, the condition is check for mobile and other platform